src/applications/Applications.SmiRunner/ServiceVerbs.cs

```diff
@@ -18,6 +18,9 @@ public abstract class ApplicationVerbBase : VerbBase
     [Verb("dicom-directory-processor", HelpText = "Queue dicom files on disk for ETL")]
     public sealed class DicomDirectoryProcessorVerb : ApplicationVerbBase { }
 
+    [Verb("extraction-launcher", HelpText = "Launch an extraction job")]
```

Comment:
Maybe just 'extract' i.e. 

```bash
./smi extract c:\myidentifiers
```

Reply:
How about `extract-images` ? I think `extract` on its own is a bit ambiguous.

Reply:
I'd be fine with `extract-images`